Utils/Dataflow/pyDKB/dataflow/stage/AbstractProcessorStage.py

```diff
@@ -309,8 +310,8 @@ def stream_input(self, fd):
             for line in iterator:
                 yield self.parseMessage(line)
         else:
-            raise NotImplementedError("Stream input is not implemented for"
-                                      " custom EOMessage marker.")
+            for line in custom_readline(fd, self.ARGS.eom):
+                yield self.parseMessage(line)
```

Comment:
Not sure, but maybe it would be better to make `iterator` variable defined depending on `eom` value, and leave common loop `for line in iterator: yield` outside the `if/else` clause?

Reply:
Got it. I think you are right. Now it looks a bit odd.